### PR TITLE
PP-5480 Add dummy card type field

### PR DESCRIPTION
**Pull request by stephencdaly**
Adds a `card_type` field in the `card_details` object in card payment responses. This will currently always return null, but in future will be implemented so that it returns `debit` or `credit` if the card type can be determined.

### CODE REVIEW

File: src/main/java/uk/gov/pay/api/model/CardDetails.java [hunk 1]
```diff
@@ -73,4 +76,9 @@ public String getExpiryDate() {
     public String getCardBrand() {
         return cardBrand;
     }
+
+    @ApiModelProperty(value = "The card type, `debit` or `credit` or `null` if not able to determine", allowableValues = "debit,credit,null", example = "debit")
```
Comment (oswaldquek) [inline src/main/java/uk/gov/pay/api/model/CardDetails.java]: how about `...or null if unknown`?

Comment (stephencdaly) [reply src/main/java/uk/gov/pay/api/model/CardDetails.java]: "if not able to determine" was what Mark and Alex decided on

Comment (alexbishop1) [reply src/main/java/uk/gov/pay/api/model/CardDetails.java]: The use of “not able to determine” comes from conversations with @m-green and will match the phrasing in the documentation.

Comment (m-green) [reply src/main/java/uk/gov/pay/api/model/CardDetails.java]: Just a heads-up that I'm likely to tweak this wording slightly, sorry!

Comment (m-green) [reply src/main/java/uk/gov/pay/api/model/CardDetails.java]: But I wouldn't let that get in the way of this PR.

Comment (stephencdaly) [reply src/main/java/uk/gov/pay/api/model/CardDetails.java]: 👍

**Pull request merged**